Add code and UAST buttons in results table

smacker opened the pull request:
Based on #38
Fix: #23

<img width="1152" alt="screen shot 2018-05-23 at 16 27 06" src="https://user-images.githubusercontent.com/406916/40430792-37142f1e-5ea6-11e8-97f2-2d25d640c434.png">
<img width="1041" alt="screen shot 2018-05-23 at 16 27 18" src="https://user-images.githubusercontent.com/406916/40430793-3734bedc-5ea6-11e8-965a-2894e9e6a9ff.png">
<img width="1131" alt="screen shot 2018-05-23 at 16 27 25" src="https://user-images.githubusercontent.com/406916/40430794-374f9a40-5ea6-11e8-8e2c-89dcfb5038e7.png">

bzz commented on frontend/src/components/ResultsTable.js:
```diff
@@ -13,8 +15,15 @@ class ResultsTable extends Component {
             return v.toString();
+          case 'string':
+            // assume any multiline string is code
+            if (v.split('\n').length > 1) {
```
Would this in JS work for both platform-dependent line endings?

Would it be possible to include a simple test for this function, so further refactoring would rely on it?

On the side-note, in case >= 1 could work for this, may be some kind of equivalent of `.indeOf > 0` could be more performant..

smacker replied:
window line ending is `\r\n` so it will work.
👍 for `indexOf`.

bzz commented:
Also, here and in other pray rebase would make review life a bit easier.

smacker commented:
changed split to indexOf.

Added simple test. Though I don't see any value in it. Current logic is completely wrong (for example commit messages can be displayed as code according to it). And I don't think it's possible to solve on frontend. But we agreed to go with this simple-wrong way for now.

carlosms commented on frontend/src/components/ResultsTable.js:
```diff
@@ -14,8 +16,15 @@ class ResultsTable extends Component {
             return v.toString();
+          case 'string':
+            // assume any multiline string is code
+            if (v.indexOf('\n') > 1) {
```
Should this be `> -1`? The way it is a multiline file with a blank line will not be detected.

smacker commented:
@bzz @carlosms please another round.
